rasa/cli/evaluate.py

```diff
@@ -120,30 +124,32 @@ def _run_markers(
         strategy: Strategy to use when selecting trackers to extract from.
         config: Path to the markers definition file to use.
         output_filename: Path to write out the extracted markers.
-        stats_file: (Optional) Path to write out statistics about the extracted
-                    markers.
+        stats_file_prefix: (Optional) Prefix of paths to write out statistics about the
```

Comment:
any suggestion for a nice name? :/

Reply:
I think the name is fine, but the docs should maybe read differently - it's a bit complicated because this is bundling two separate things in, a path, and the name prefix. Maybe something like "The path where stats files should be stored, and a prefix used to construct the stats file names" or something?

Reply:
Tried to add that to both the docstr and the help text with an example.  A bit lengthy now but should be clearer now?